test/mitmproxy/tools/web/test_app.py

```diff
@@ -403,3 +404,72 @@ def test_websocket(self):
         # trigger on_close by opening a second connection.
         ws_client2 = yield websocket.websocket_connect(ws_url)
         ws_client2.close()
+
+
+class TestProcessList(tornado.testing.AsyncHTTPTestCase):
```

Comment:
Looks good, but do we need to mock here? We can easily avoid that here, we don't have to assert anything specific about the processes. I was thinking of just having a test like this:

https://github.com/mitmproxy/mitmproxy/blob/b3a3f43bd10b23f3911d015eb4633e5cb9580d5a/test/mitmproxy/tools/web/test_app.py#L344-L347

with just `assert get_json(resp)` in the last line.